python/lsst/ts/electrometer/enums.py

```diff
@@ -22,7 +22,7 @@
 import enum
 
 
-class UnitMode(str, enum.Enum):
+class UnitMode(enum.StrEnum):
```

Comment:
Please revert this change and the one in line 50. The code needs to be backward compatible with Python 3.10.